README.md

```diff
@@ -284,6 +284,9 @@ Make sure no one is logged in
 #### `Guardian.Plug.LoadResource`
 If a token was found, load the resource for it
 
+#### `Guardian.Plug.EnsureResource`
+Make sure that current resource has been set, usually in `Guardian.Plug.LoadResource`
```

Comment:
Should this read: "Make sure _the_ current resource has been set" instead of "that?"